src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -0,0 +1,146 @@
+// Node modules.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { isEmpty } from 'lodash';
+// Relative imports.
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import environment from 'platform/utilities/environment';
+import { apiRequest } from 'platform/utilities/api';
+
+class CernerCallToAction extends Component {
+  static propTypes = {
+    type: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
+    // From mapStateToProps.
+    facilities: PropTypes.arrayOf(
+      PropTypes.shape({
+        facilityId: PropTypes.string.isRequired,
+        isCerner: PropTypes.bool.isRequired,
+      }).isRequired,
+    ).isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: '',
+      facilities: [],
+      fetching: true,
+    };
+  }
+
+  componentDidMount() {
+    const { facilities } = this.props;
+
+    // Escape early if there are no facilities.
+    if (isEmpty(facilities)) {
+      return;
+    }
+
+    // Derive the cerner facilities.
+    const cernerFacilities = facilities.filter(facility => facility.isCerner);
+
+    // Escape early if there are no cerner facilities.
+    if (isEmpty(cernerFacilities)) {
+      // WARNING: Add sentry logging here if there are no cerner facilities found, this should never happen as the component only renders when there ARE cerner facilities.
+      return;
+    }
+
+    // Derive the list of facility IDs.
+    const cernerFacilityIDs = cernerFacilities.map(
+      facility => `vha_${facility.facilityId}`,
+    );
+
+    // Fetch cerner facilities.
+    this.fetchFacilities(cernerFacilityIDs);
```

Comment:
The whole API layer of this could make a cool shared action for `fetchUserFacilities` or something like that, if we find that we need it again

Reply:
Definitely, it actually doesn't use redux at all atm apart from getting the facilities objects from authd-xp, ideally we would just include the facility name + location as part of the `/user` call long-term I think